src/api/devices.rs

```diff
@@ -318,3 +321,55 @@ impl Command<AuthenticateCommand> {
         Ok(())
     }
 }
+
+#[derive(Parser, Debug)]
+pub struct GetUpdateCommand {
+    /// The PRN of the device you wish to create the resource within.
+    #[arg(
+        long,
+        value_parser = PRNValueParser::new(PRNType::Device)
+    )]
+    device_prn: String,
+
+    /// The PRN of the device's current release.
```

Comment:
Is there any way to require at least one of the additional args? Omitting the device prn arg is handled by the CLI. If the user doesn't add anything else, you'll get an API response whose error message. 

```
./target/release/peridio-cli --profile local-lee devices get-update --device-prn prn:1:ef179816-38ec-412a-b8bf-7391a76b033c:device:ba09626f-1099-4edd-af3c-6223a2e761f7
{"code":50,"data":{"params":{"release_prn":["One or a combination of the release_prn, bundle_prn, or release_version params must be provided."]},"body":[],"headers":{}},"type":"bad_request"}
```

Maybe it's a pain to do a "at least one of these" validation..